app/build.gradle

```diff
@@ -54,6 +55,7 @@ android {
             buildConfigField "String", "LOGIN_BASE_URL", "\"https://permanent-dev.fusionauth.io/oauth2/\""
             buildConfigField "String", "BASE_URL", "\"https://staging.permanent.org/\""
             buildConfigField "String", "BASE_API_URL", "\"https://staging.permanent.org/api/\""
+            buildConfigField "String", "BASE_API_URL_NEW", "\"https://api.staging.permanent.org/\""
```

Comment:
I think the name of this API is Stela. Maybe rename it to something like BASE_API_URL_STELA